doc/rst/technotes/libraries.rst

```diff
@@ -384,6 +416,69 @@ For example:
    Make sure your Python functionality is also complete before calling this
    function.
 
+.. note::
+
+  If you are taking advantage of the generated ``__init__.py`` initializer
+  file to import the output directory as a package, you do not need to call
+  ``chpl_cleanup()`` yourself because it is already registered to be called
+  at program exit. The generated initializer is explained below.
+
+.. _Python_Init_File:
+
+Python Init File
+----------------
+
+A simple ``__init__.py`` file is generated in the output directory along with
+the Python module. It looks roughly like the following:
+
+.. code-block:: python
+
+  import atexit
+
+  #
+  # Here directoryName is the name of the directory containing your Python
+  # module, and moduleName is the name of your Python module.
+  #
+  from directoryName.moduleName import *
+
+  atexit.register(moduleName.chpl_cleanup)
+
+The initializer file transforms the output directory into a Python package.
+It will also register ``chpl_cleanup()`` to be called automatically at
+program exit.
+
+The directory containing generated Python code must be visible to any script
```

Comment:
I think our attempt to summarize the python import system makes this paragraph harder to follow than is has to be. Maybe we could just list the most common approaches (importing locally or through `PYTHONPATH`) and then link the python docs for more details, e.g.

```
Like any other Python package, the generated Python package must be visible 
in order to import it, such as through importing it locally or adding the path to ``PYTHONPATH``.
Refer to the `Python import documentation`_ for more details on python import visibility.

.. _Python import documentation https://docs.python.org/3/reference/import.html#the-import-system
```